plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Assets.js

```diff
@@ -162,6 +168,49 @@ Ext.define('NX.coreui.controller.Assets', {
         assetContainer = me.getAssetContainer();
 
     assetContainer.refreshInfo(gridView.up('grid').componentModel, assetModel);
+  },
+
+  /**
+   * @protected
+   * Enable 'Delete' when user has 'delete' permission.
+   */
+  bindDeleteAssetButton: function(button) {
+    var me = this,
+        component = me.getAssetList().componentModel;
+    button.mon(
+        NX.Conditions.or(
+            NX.Conditions.isPermitted('nexus:repository-admin:*:*:delete'),
+            NX.Conditions.isPermitted('nexus:repository-admin:' + component.get('format') + ':' +
+                component.get('repositoryName') + ':delete')
```

Comment:
@jdillon This seemed rational to me, but I defer to you for exactly which perms should be here. As to the implied permission, I am OR'ing to see if the user has blanked admin perms on all repos OR just for this repo. I imagine the case would be the same with a change to repository-view:*:*:delete